src/routes/GameScreen.js

```diff
@@ -33,6 +32,19 @@ export default class GameScreen extends Component {
         winnerId: -1,
     };
 
+    waitForOtherPlayer = async (id, token) => {
+        const { otherPlayer } = await game()(id, token);
+        if (otherPlayer) {
+            return Promise.resolve(otherPlayer);
+        }
+
+        await new Promise(resolve => {
+            setTimeout(() => resolve(), config.refreshDuration);
+        });
+
+        return this.waitForOtherPlayer(id, token);
```

Comment:
So we wait indefinitely for another player ?

Reply:
Yes, if you don't want to wait, you can go back and play a single game.

It's possible to add feedbacks or a direct button to create a single game for example.